lms/static/js/student_account/views/RegisterView.js

```diff
@@ -56,12 +55,14 @@
                             currentProvider: this.currentProvider,
                             providers: this.providers,
                             hasSecondaryProviders: this.hasSecondaryProviders,
-                            themeName: this.themeName,
                             privacyPolicyUrl: this.privacyPolicyUrl,
                             platformName: this.platformName
                         }
                     }));
 
+                    var regExtra = $('#register-extra-tpl').html();
```

Comment:
If you still want to make this a one-liner, I'll look the other way ;)

Reply:
nah, I'm cool as is.